src/Spice86/MemoryWrappers/MemoryReadOnlyBitRangeUnion.cs

```diff
@@ -9,15 +9,17 @@
 using System.Collections.Specialized;
 
 public class MemoryReadOnlyBitRangeUnion : IReadOnlyBitRangeUnion {
-    private readonly IMemory _memory;
+    private readonly uint _startAddress;
+    private readonly uint _endAddress;
 
-    public MemoryReadOnlyBitRangeUnion(IMemory memory) {
-        _memory = memory;
-        EnclosingRange = new BitRange(0, _memory.Length * 8);
+    public MemoryReadOnlyBitRangeUnion(uint startAddress, uint endAddress) {
```

Comment:
A comment that states that the endAddress is not included in the range would be appreciated.

Reply:
Implemented as XML documentation.